libmapi/simple_mapi.c

```diff
@@ -184,6 +186,35 @@ static enum MAPISTATUS CacheDefaultFolders(mapi_object_t *obj_store)
 	retval = GetFIDFromEntryID(entryid->cb, entryid->lpb, id_inbox, &store->fid_drafts);
 	OPENCHANGE_RETVAL_IF(retval, retval, mem_ctx);
 
+	/* [MS-OXOSFLD] Section 2.2.4 */
+	entries_ids = (const struct BinaryArray_r *)find_SPropValue_data(&aRow, PR_ADDITIONAL_REN_ENTRYIDS);
+	OPENCHANGE_RETVAL_IF(!entries_ids, MAPI_E_NOT_FOUND, mem_ctx);
+
+	/* set cached Sync Issues FID */
+	entryid = entries_ids->lpbin + 1;
+	retval = GetFIDFromEntryID(entryid->cb, entryid->lpb, id_inbox, &store->fid_sync_issues);
+	OPENCHANGE_RETVAL_IF(retval, retval, mem_ctx);
```

Comment:
Personally I'd prefer explicit `retval != MAPI_E_SUCCESS` but i can live without that